EssentialsProtect/src/com/earth2me/essentials/protect/EssentialsProtect.java

```diff
@@ -52,14 +51,19 @@ private void initialize(final PluginManager pm, final Plugin essPlugin) {
         pm.registerEvents(entityListener, this);
 
         if (VersionUtil.getServerBukkitVersion().isHigherThan(VersionUtil.v1_13_2_R01)) {
-            final EssentialsProtectEntityListener1_13_2_R1 entityListener1_13_2_r1 = new EssentialsProtectEntityListener1_13_2_R1(this);
+            final EssentialsProtectEntityListener_1_13_R1 entityListener1_13_2_r1 = new EssentialsProtectEntityListener_1_13_R1(this);

```

Comment:
```suggestion
            final EssentialsProtectEntityListener_1_13_2_R1 entityListener1_13_2_r1 = new EssentialsProtectEntityListener_1_13_2_R1(this);

```

reeeeeee

*(technically this listener only applies to 1.13.2+, see #2942)*

Reply:
Forgot about that, sure. Was just noticing a naming inconsistency between the files and it annoyed me :P

Reply:
OK fixed